Chirp/Pages/ChatsPage/ChatsTableViewCell.swift

```diff
@@ -0,0 +1,128 @@
+    }()
+    
+    // Setup tableview cell
+    func setup() {
+        setupViews()
+        setupConstraints()
+    }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        friendImageView.layer.cornerRadius = friendImageView.frame.width/2
+        unreadMessageCounterLabel.layer.cornerRadius = unreadMessageCounterLabel.frame.width/2
+        
+    }
+    
+    // Add subviews
+    private func setupViews() {
+        addSubview(cellBoundaryview)
+        addSubview(friendImageView)
+        addSubview(fullNameLabel)
+        addSubview(lastMessageLabel)
+        addSubview(lastMessageTimeLabel)
+        addSubview(unreadMessageCounterLabel)
+    }
+    
+    // Configure constraints for the subviews
+    private func setupConstraints() {
+        NSLayoutConstraint.activate([
+            cellBoundaryview.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 24),
+            cellBoundaryview.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -24),
+            cellBoundaryview.topAnchor.constraint(equalTo: topAnchor),
+            cellBoundaryview.bottomAnchor.constraint(equalTo: bottomAnchor),
+            cellBoundaryview.heightAnchor.constraint(greaterThanOrEqualToConstant: imageViewWidth + 20),
+            
+            friendImageView.leadingAnchor.constraint(equalTo: cellBoundaryview.leadingAnchor),
+            friendImageView.centerYAnchor.constraint(equalTo: cellBoundaryview.centerYAnchor),
+            friendImageView.widthAnchor.constraint(equalToConstant: imageViewWidth),
+            friendImageView.heightAnchor.constraint(equalToConstant: imageViewWidth),
+            
+            fullNameLabel.leadingAnchor.constraint(equalTo: friendImageView.trailingAnchor, constant: leadingTrailingAnchor),
+            fullNameLabel.heightAnchor.constraint(equalToConstant: labelHeight),
+            fullNameLabel.topAnchor.constraint(equalTo: friendImageView.topAnchor, constant: leadingTrailingAnchor),
+            
+            lastMessageTimeLabel.trailingAnchor.constraint(equalTo: cellBoundaryview.trailingAnchor),
+            lastMessageTimeLabel.heightAnchor.constraint(equalToConstant: labelHeight),
+            lastMessageTimeLabel.topAnchor.constraint(equalTo: fullNameLabel.topAnchor),
+            
+            lastMessageLabel.leadingAnchor.constraint(equalTo: friendImageView.trailingAnchor, constant: leadingTrailingAnchor),
+            lastMessageLabel.heightAnchor.constraint(equalToConstant: labelHeight),
+            lastMessageLabel.bottomAnchor.constraint(equalTo: friendImageView.bottomAnchor, constant: -leadingTrailingAnchor),
+            
+            unreadMessageCounterLabel.trailingAnchor.constraint(equalTo: cellBoundaryview.trailingAnchor),
+            unreadMessageCounterLabel.heightAnchor.constraint(equalToConstant: labelHeight),
+            unreadMessageCounterLabel.bottomAnchor.constraint(equalTo: friendImageView.bottomAnchor, constant: -leadingTrailingAnchor)
```

Comment:
This is a solution that works, but in order to have a more solid design, please consider using stackViews to more easily manage your constraints especially when you have many views connecting to themselves.
For example, for your fullNameLabel, you assigned the following constraints (leading, height, top) if the name is too long, this will overlap with the time label. Also because of the height constraint, if we decide to change the font size or enable dynamic font for accessibility, this will break.

My Suggestion considering your layout:
1. Create a horizontal stack (stack1)
2. Create two vertical stacks (stack2, stack3)
3. Inside stack2, insert fullName and lastMessage
4. Inside stack3, insert timeLbl, unreadCount. Also set the stack3 alignment to .center
5. Inside stack 1, insert profileImage, stack2, stack3

If this does not make sense please let me know and we can tackle it together on a call.